operations/companies.md

```diff
@@ -244,36 +326,196 @@ Adds a new company. Note this operation supports [Portfolio Access Tokens](../gu
-| `ExternalIdentifier` | string | optional, max 255 characters | Identifier of the company from external system. |
 | `ReferenceIdentifier` | string | optional | External system identifier - custom identifier used by an external system such as an external database. |
-| `WebsiteUrl` | string | optional | The website url of the company. |
+| `AccountingCode` | string | optional | Accounting code of the company. |
+| `AdditionalTaxIdentifier` | string | optional | Additional tax identifier of the company. |
+| `BillingCode` | string | optional | Billing code of the company. |
+| `Contact` | string | optional | Other contact details, such as telephone, email or similar. |
+| `ContactPerson` | string | optional | Contact person of the company. |
+| `ElectronicInvoiceIdentifier` | string | optional | Electronic invoice identifier of the company. |
+| `Identifier` | string | optional | Other identifier of the company, e.g. legal identifier. |
+| `Iata` | string | optional | Iata of the company. |
+| `IsActive` | boolean | required | Whether the company is still active. |
+| `Notes` | string | optional | Additional notes. |
+| `Number` | integer | required | Unique number of the company. |
+| `TaxIdentifier` | string | optional | Tax identification number of the company. |
+| `Telephone` | string | optional | Contact telephone number. |
+| `CreatedUtc` | string | optional | Date of [Company](companies.md#company) creation date and time. |
+| `UpdatedUtc` | string | optional | Date of [Company](companies.md#company) last update date and time. |
+| `AddressId` | string | optional | Unique identifier of the company [Address](addresses.md#account-address). |
+| `MergeTargetId` | string | optional |  |
+| `ExternalIdentifier` | string | optional, max length 255 characters | Identifier of company from external system. |
+| `IsUpdatedByMe` | boolean | optional |  |
+| ~~`Address`~~ | ~~[OldAddress](configuration.md#address)~~ | ~~optional~~ | ~~Address of the customer.~~ **Deprecated!** |
+| ~~`TaxIdentificationNumber`~~ | ~~string~~ | ~~optional~~ | **Deprecated!** |
 
-#### Company options parameters
+#### CompanyOptions
+Options of the company.
 
 | Property | Type | Contract | Description |
 | :-- | :-- | :-- | :-- |
-| `Invoiceable` | boolean | required | Whether the company is invoiceable or not. |
-| `AddFeesToInvoices` | boolean | required | Whether the company has an additional fee applied for invoicing or not. |
+| `Invoiceable` | boolean | required |  |
+| `AddFeesToInvoices` | boolean | required |  |
+| `AddTaxDeductedPaymentToInvoices` | boolean | required |  |
 
-### Response
+#### CreditRating
+Credit rating to define creditworthiness of the company.
 
-Same structure as in [Get all companies](#get-all-companies) operation.
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Basic` | string | optional |  |
```

Comment:
❓ Question: is it really optional? and what description should I add?

Reply:
Looks like this one was added for Strawberry (NCH) specifically and to make it worse, it's supposed to be an enum. There's some description for it in https://github.com/MewsSystems/mews/pull/35854

> CreditRating.Basic has three valid values:
>
>    "CreditOk"
>    "PaymentRequiredUpfront"
>    "LocalDecisionRequired"

I don't know much about that, so I recommend asking in `rnd-accounts` with link to that PR.

Reply:
@jnv should I change it to enum as a part of this task?